docs/05-internals/design/api-v2/gravsearch.md

```diff
@@ -303,13 +298,13 @@ using `UNION` statements for all subclasses and subproperties from the ontologie
 Similarly, the API replaces `knora-api:standoffTagHasStartAncestor` with `knora-base:standoffTagHasStartParent*`.
 
 
-# Optimisation of generated SPARQL
+## Optimisation of generated SPARQL
```

Comment:
This is a mistake, too. This (and the following) title were not displayed in the nav bar, because they break markdown rules.